### DISCUSSION TITLE
Smart Connections + Dataview

### DISCUSSION THREAD

**Discussion by aaroneden** [brianpetro/obsidian-smart-connections#161] [Q&A]
When SmartConnections processes my files and creates the vector embeddings database.  Does it do this with the Rendered files or source files?

Meaning -- If I have a dataview in one of my files, will the embedding include the results of the dataview query?

**Comment by brianpetro** [comment]
Hey @aaroneden 

Currently, only the Dataview query, not the results, are included in the embeddings.

The only place the Dataview queries are rendered is when you use a specific note as context using the [[link]] syntax in the Smart Chat, currently.

I'm working towards embedding Dataview code blocks, once as query and once as results. The former so that the Smart Chat may generate new queries to render as context. 

Let me know if that clears things up for you, 
Brian 🌴

**Comment by aaroneden** [comment]
Yes, that does clear it up, thanks!
